.github/ISSUE_TEMPLATE/bug_report.yml

```diff
@@ -73,4 +73,4 @@ body:
 - type: markdown
   attributes:
     value: >
-      Thanks for helping us improve the library!
```

Comment:
please revert :)

Reply:
Don't why, it's pre-commit which did that

Reply:
I see my pre-commit was out of date (after autoupdate it does the same ^^)